specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/virtualWan.json

```diff
@@ -2276,6 +2411,149 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnGateways/{gatewayName}/vpnConnections/{vpnConnectionName}startpacketcapture": {
```

Comment:
since this is modeled as a POST, it cant be invoked from an aRM template. Hope that is ok for customers. Else, it could be modeled as a PUT.

Reply:
Yes, here is the link https://docs.microsoft.com/en-us/rest/API/network-gateway/virtualnetworkgateways/startpacketcapture it is the same thing ,